sympy/physics/_biomechanics/musculotendon.py

```diff
@@ -395,7 +705,51 @@ def _fiber_length_explicit_musculotendon_dynamics(self):
 
     def _tendon_force_explicit_musculotendon_dynamics(self):
         """Elastic tendon musculotendon using `F_T_tilde` as a state."""
-        raise NotImplementedError
+        self._F_T_tilde = dynamicsymbols(f'F_T_tilde_{self.name}')
+        self._l_MT = self.pathway.length
+        self._v_MT = self.pathway.extension_velocity
+        self._fl_T = self._F_T_tilde
+        if self._with_defaults:
+            self._fl_T_inv = self.curves.tendon_force_length_inverse.with_defaults(self._fl_T)
+        else:
+            fl_T_constants = symbols('c_0:4_fl_T')
+            self._fl_T_inv = self.curves.tendon_force_length_inverse(self._fl_T, *fl_T_constants)
+        self._l_T_tilde = self._fl_T_inv
+        self._l_T = self._l_T_tilde*self._l_T_slack
+        self._l_M = sqrt((self._l_MT - self._l_T)**2 + (self._l_M_opt*sin(self._alpha_opt))**2)
+        self._l_M_tilde = self._l_M/self._l_M_opt
+        if self._with_defaults:
+            self._fl_M_pas = self.curves.fiber_force_length_passive.with_defaults(self._l_M_tilde)
+            self._fl_M_act = self.curves.fiber_force_length_active.with_defaults(self._l_M_tilde)
+        else:
+            fl_M_pas_constants = symbols('c_0:2_fl_M_pas')
+            self._fl_M_pas = self.curves.fiber_force_length_passive(self._l_M_tilde, *fl_M_pas_constants)
+            fl_M_act_constants = symbols('c_0:12_fl_M_act')
+            self._fl_M_act = self.curves.fiber_force_length_active(self._l_M_tilde, *fl_M_act_constants)
+        self._cos_alpha = (self._l_MT - self._l_T)/self._l_M
+        self._F_T = self._F_T_tilde*self._F_M_max
+        self._F_M = self._F_T/self._cos_alpha
+        self._F_M_tilde = self._F_M/self._F_M_max
+        self._fv_M = (self._F_M_tilde - self._fl_M_pas)/(self.a*self._fl_M_act)
+        if self._with_defaults:
+            self._fv_M_inv = self.curves.fiber_force_velocity_inverse.with_defaults(self._fv_M)
+        else:
+            fv_M_constants = symbols('c_0:4_fv_M')
+            self._fv_M_inv = self.curves.fiber_force_velocity_inverse(self._fv_M, *fv_M_constants)
+        self._v_M_tilde = self._fv_M_inv
+        self._v_M = self._v_M_tilde*self._v_M_max
+        self._v_T = self._v_MT - (self._v_M/self._cos_alpha)
+        self._v_T_tilde = self._v_T/self._l_T_slack
+        if self._with_defaults:
+            self._fl_T = self.curves.tendon_force_length.with_defaults(self._l_T_tilde)
+        else:
+            fl_T_constants = symbols('c_0:4_fl_T')
+            self._fl_T = self.curves.tendon_force_length(self._l_T_tilde, *fl_T_constants)
+        self._dF_T_tilde_dt = self._fl_T.diff(dynamicsymbols._t).subs({self._l_T_tilde.diff(dynamicsymbols._t): self._v_T_tilde})
+
+        self._state_vars = Matrix([self._F_T_tilde])
+        self._input_vars = zeros(0, 1)
+        self._state_eqns = Matrix([self._dF_T_tilde_dt])
```

Comment:
No idea how to check this stuff? Are we adding a test simulation somewhere to verify it works? Or do you want me to read the paper and check your math equations?

Reply:
When I wrote the implementation, I first hand-derived the equations myself using only the basic musculotendon equations from the paper and knowledge of which entity to choose as the state variable. I then implemented the equations following the explanation from the paper's supplementary material in the test module. Where the two didn't match, highlighted by the tests failing, I went back and double checked both methods. I found two mistakes (I'd missed a `cos(alpha)` term when translating from my notebook to code, and I'd mistakenly used a regular curve when the inverse curve was required). After fixing these the tests then passed. I obviously can't guarantee that they're 100% correct, but by implementing them two separate ways and getting the same answers I have high confidence that they are correct. You are welcome to check the maths yourself, but I have already done this pretty extensively.

Reply:
I know/trust you've done this carefully and doing it two ways is superb, but the only way I've figured out how to know 100% I'm right with these kinds of things in the past is to evaluate models numerically with random inputs and compare the results to someone else's derivation for the same random inputs. Checking whether complex math symbols match is error prone. If you desire a true review of these equations, then we need to think about how to do that. Otherwise we merge and see if anyone in the future ever complains :)

Reply:
Okay, in that case I agree, a comparison against OpenSim would do this thoroughly. I vote we merge this PR and I open an issue about validating these against the OpenSim version of these models numerically. If I have time before the end of the month to do this myself then I will, but if I can't get round to it before then then at least that issue will act as evidence that there's the possibility of bugs if users encounter weird behaviour.

Reply:
Opening the issue if enough, doing the validation should be low on the priority for the final tasks.